src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {

```

Comment:
When creating a new Square instance, it's better to avoid passing expressions directly in the constructor. Instead, create a variable for the 'side' value and then pass it to the constructor for better readability.